src/vrs_anvil/cli.py

```diff
@@ -251,7 +250,7 @@ def ps_cli(ctx):
                     pass
 
                 click.secho(
-                    f"🚧  pid: {str(process['pid'])}, manifest: {str(process['manifest'])}, vcf: {str(process['vcf'])}, metrics_file: {metrics_file}, log_file: {log_file}",
+                    f"🚧  pid: {process['pid']!s}, manifest: {process['manifest']!s}, vcf: {process['vcf']!s}, metrics_file: {metrics_file}, log_file: {log_file}",
```

Comment:
had to GPT this syntax lol

Reply:
I think we can remove the `!s` since the f string already converts the variables into a string